core/models.py

```
@@ -182,3 +182,50 @@ class HomepageSettings(BaseGenericSetting):
+        default=False,
+        help_text=(
+            "Pin this term above Algolia-derived rows on the homepage. "
+            "Use rank to order among multiple pins."
+        ),
+    )
+    updated_at = models.DateTimeField(auto_now=True)
+
+    objects = PopularSearchTermManager()
+
+    class Meta:
+        ordering = ["-is_pinned", "rank", "label"]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)